frontend/e2e-tests/dom-to-db-tests/resume-data-entry.e2e.ts

```diff
@@ -75,11 +75,6 @@ test.describe("resume data entry flow", () => {
       await expect(votersAndVotesPage.blankVotesCount).toBeEmpty();
       await expect(votersAndVotesPage.invalidVotesCount).toBeEmpty();
       await expect(votersAndVotesPage.totalVotesCastCount).toHaveValue("100");
-
-      await votersAndVotesPage.abortInput.click();
-      await expect(abortInputModal.heading).toBeFocused();
-      await abortInputModal.close.click();
-      await expect(votersAndVotesPage.abortInput).toBeFocused();
```

Comment:
The modal should still get a focused heading right? @jorisleker  I think you cannot tab through it otherwise

Reply:
Yes, and it the h2 still gets focussed. So see no reason to remove this test.

Reply:
The test was removed here because the subject of the test is "resume after saving" and tests should not do too much (ideally only one thing). Modal will still get a focused heading and this is also already checked in tests on six other locations.

The last line was the trigger actually, because the abortInput will not receive focus again after closing the modal in Safari only.